### ISSUE TITLE
Hey How can i get the code train.py of this model

### ISSUE THREAD

**Issue by UCASSMY** [linyq2117/SAMRefiner#1]
Hey How can i get the code train.py of this model

**Comment by linyq2117** [comment]
Hi, thanks for your interest.

Our SAMRefiner is a training-free method based on the pre-trained SAM. You can directly use it to refine coarse masks. SAMRefiner++ (SAMRefiner + IoU Adaption) is proposed to exploit the prior knowledge in coarse masks, thus having prerequisites on the quality of coarse masks. Most experiments in the paper are based on the training-free SAMRefiner while SAMRefiner++ is only explored on DAVIS-585. We believe the direct SAMRefiner can perform well in most cases and we may consider releasing the code of SAMRefiner++ in the next few days.

**Comment by linyq2117** [comment]
Sorry for the delay, we plan to release this part of the code before the conference starts.

**Comment by linyq2117** [comment]
The code for SAMRefiner++ has been released at [SAMRefiner_plus](https://github.com/linyq2117/SAMRefiner/tree/main/SAMRefiner_plus). Feel free to try it.

**Comment by LZH163-Lab** [comment]
demo可以批量处理一个文件夹里的图片吗，不是单张这种

**Comment by linyq2117** [comment]
Closed for inactivity.
